open_ce/conda_utils.py

```diff
@@ -72,3 +75,27 @@ def get_output_file_paths(meta, variants):
         result.append(os.path.join(path.parent.name, path.name))
 
     return result
+
+def conda_package_info(channels, package):
+    '''
+    Get conda package info.
+    '''
+    pkg_args = "\"{}\"".format(generalize_version(package))
+    channel_args = " ".join({"-c \"{}\"".format(channel) for channel in channels})
+
+    cli = "conda search --json {} {} --info".format(channel_args, pkg_args)
+    ret_code, std_out, _ = run_command_capture(cli, stderr=subprocess.STDOUT)
+    if not ret_code:
+        raise OpenCEError(Error.CONDA_PACKAGE_INFO, cli, std_out)
+    return std_out
+
+def get_latest_package_info(channels, package):
+    '''
+    Get the conda package info for the most recent search result.
+    '''
+    results = yaml.safe_load(conda_package_info(channels, package))
```

Comment:
Doesn't `conda_package_info` return output from a JSON file? If so, should probably use `json.loads`.

Reply:
I saw a recommendation somewhere to use yaml safe load even for json. Let me see if I can find the reasoning.

Reply:
Ah OK. I'm guessing due to the "safe" part.

Reply:
I've switched this to json load to be more precise.